File: Java/Mhs.java

//import library
import java.util.ArrayList;
import java.util.Scanner;

// class mahasiswa berisi data mahasiswa berupa nama, nim, prodi, dan fakultas
public class Mhs {
    // atribut private
    private String nama;
    private String nim;
    private String prodi;
    private String fakultas;

    // atribut public

    // Constructor untuk mengubah value menjadi default
    public Mhs() {
        this.nama = "";
        this.nim = "";
        this.prodi = "";
        this.fakultas = "";
    }

    // public method
    // setter dan getter
    public String getNama() {
        return this.nama;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public String getNim() {
        return this.nim;
    }

    public void setNim(String nim) {
        this.nim = nim;
    }

    public String getProdi() {
        return this.prodi;
    }

    public void setProdi(String prodi) {
        this.prodi = prodi;
    }

    public String getFakultas() {
        return this.fakultas;
    }

    public void setFakultas(String fakultas) {
        this.fakultas = fakultas;
    }

}

File: PHP/Mhs.php
<?php

// class mahasiswa berisi data mahasiswa berupa nama, nim, prodi, dan fakultas
class Mhs
{
    //private
    private $nama = '';
    private $nim = '';
    private $prodi = '';
    private $fakultas = '';
    private $foto = '';
    //public
    //construct
    public function __construct($nama = '', $nim = '', $prodi = '', $fakultas = '', $foto = '')
    {
        $this->nama = $nama;
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
        $this->foto = $foto;
    }
    //method setter dan getter
    public function setnama($nama)
    {
        $this->nama = $nama;
    }
    public function setnim($nim)
    {
        $this->nim = $nim;
    }
    public function setprodi($prodi)
    {
        $this->prodi = $prodi;
    }
    public function setfakultas($fakultas)
    {
        $this->fakultas = $fakultas;
    }
    public function setfoto($foto)
    {
        $this->foto = $foto;
    }

    public function getnama()
    {
        return $this->nama;
    }
    public function getnim()
    {
        return $this->nim;
    }
    public function getprodi()
    {
        return $this->prodi;
    }
    public function getfakultas()
    {
        return $this->fakultas;
    }
    public function getfoto()
    {
        return $this->foto;
    }
    //destructor
    function __destruct()
    {
    }
}
